src/Uno.UI.Runtime.Skia.Gtk/Input/GtkCorePointerInputSource.cs

```diff
@@ -486,7 +486,7 @@ private void UseDevice(PointerPoint pointer, Gdk.Device device)
 
 		var pointerPoint = new Windows.UI.Input.PointerPoint(
 			frameId: time,
-			timestamp: time,
+			timestamp: time * (ulong)TimeSpan.TicksPerMillisecond, // time is in ms, timestamp is in ticks
```

Comment:
@dr1rrb do you have an opinion on this?

Reply:
Yes I have an opinion : we should do the same fix with "FrameBuffer" (for mouse and touch) where the timestamp is in micro sec instead of ticks :) (According to the doc of `libinput_event_pointer_get_time_usec` and `libinput_event_touch_get_time_usec`)

BTW sounds unrelated to the current PR, should we move those fixes to another one?

Reply:
Edit: just searched the doc for the timestamp: "The time, relative to the system boot time, in microseconds." (https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.input.pointerpoint.timestamp?view=windows-app-sdk-1.4#microsoft-ui-input-pointerpoint-timestamp)

So it means that here you should do
```suggestion
			timestamp: time * 1000, // time is in ms, timestamp is in micro-seconds
```

But it also means that we need to fix `WPFCoreInputSource`, and the gesture recognizer assumes that the timestmp is in ticks (e.g. [for long press](https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/UI/Input/GestureRecognizer.Gesture.cs#L392)) But then it's a complete different issue https://github.com/unoplatform/uno/issues/14535

Reply:
Note 2: as an intermediate solution (i.e. before we fix #14535) we should patch it by converting it to ticks on GTK and FrameBuffer, at least it will be consistent with other platforms and fix some interaction issues (hold and inertia for instance).